lib/init/config-initializer.js

```diff
@@ -483,15 +493,20 @@ function promptUser() {
             if (earlyAnswers.styleguide === "airbnb" && earlyAnswers.framework !== "react") {
                 earlyAnswers.styleguide = "airbnb-base";
             }
+            if (earlyAnswers.styleguide === "xo-typescript") {
+                earlyAnswers.styleguide = ["xo", "xo-typescript"];
```

Comment:
It seems that `eslint-config-xo-typescript` sets typescript config (parser, rules...) for all files, so the linting of `.eslintrc.js` fails. Should we maybe extend `xo-typescript` in overrides for `files: ['*.ts', '*.tsx']`, or ask them to update the config to move everything into overrides? (that would probably solve https://github.com/xojs/eslint-config-xo-typescript/issues/39, too).

Reply:
I've moved it to `config.overrides`: https://github.com/eslint/create-config/pull/33/commits/211a8f19f72dc09ce2f91f609258415be8bd3e2c